sdk/agrifood/Azure.Verticals.AgriFood.Farming/src/Generated/InsightAttachments.cs

```diff
@@ -130,6 +131,7 @@ public virtual Response CreateOrUpdate(string partyId, string modelId, string re
             Argument.AssertNotNullOrEmpty(resourceType, nameof(resourceType));
             Argument.AssertNotNullOrEmpty(resourceId, nameof(resourceId));
             Argument.AssertNotNullOrEmpty(insightAttachmentId, nameof(insightAttachmentId));
+            Argument.AssertNotNull(content, nameof(content));
```

Comment:
is this expected?
It might be intentional that we generate this content as required parameter but nullable to represent the case that a body is optional. Could you please confirm?

Reply:
This content consumes multipart/form-data. Do we agree multipart/form-data must be required even it is optional in swagger?

Reply:
If one of the components of multipart is required, `RequestContent` parameter in protocol method must be required as well. It is a good question, however, should we have required body parts at all.